### DISCUSSION TITLE
我不知道 IMESharp 是否可以用來搭配 LibvChewingNT 引擎製作輸入法。

### DISCUSSION THREAD

**Discussion by ShikiSuen** [ryancheung/ImeSharp#16] [Ideas]
> LibvChewingNT 是一款跨平台輸入法總成模組，以 .NET 6 驅動。目前正在研發。該產品的定位與 libchewing 相同，仍需在各個平台單獨開發「使用了該模組」的輸入法，畢竟每個系統平台下的輸入法體系都是不同的。作者對 Windows TSF 的理解幾乎趨近於零，所以無法獨自完成 Windows 版的威注音輸入法研發。但為了向世間拿出誠意，所以才完成了上述部分。這個倉庫就曬出來給有志者們提供一些柴禾。

https://github.com/vChewing/LibvChewingNT

**Comment by ryancheung** [comment]
应该不行。ImeSharp只是封装了输入法API帮助应用程序方便使用Windows系统输入法，它应该不能帮助实现一个输入法。

**Reply by ShikiSuen** [reply]
感謝您的解惑。看來我得找其他的解決方案了。
